src/main/scala/com/fulcrumgenomics/fastq/FastqToBam.scala

```diff
@@ -157,6 +158,7 @@ class FastqToBam
     try {
       val subs = fqs.iterator.zip(structures.iterator).flatMap { case(fq, rs) => rs.extract(fq.bases, fq.quals) }.toIndexedSeq
       val sampleBarcode = subs.iterator.filter(_.kind == SampleBarcode).map(_.bases).mkString("-")
+      val smpQual       = subs.iterator.filter(_.kind == SampleBarcode).map(_.quals).mkString(" ")
```

Comment:
1. updated the name as per above
2. Assuming you'll use `QT` as per [the SAM spec](https://samtools.github.io/hts-specs/SAMtags.pdf), in which case we want it hyphen delimited:
> The recommended implementation concatenates all the barcodes and places a hyphen (‘-’) between the barcodes from the same template.
```suggestion
      val sampleBarcodeQualTag = subs.iterator.filter(_.kind == SampleBarcode).map(_.quals).mkString("-")
```